### DISCUSSION TITLE
Error: Config file 'mkdocs.yml' does not exist

### DISCUSSION THREAD

**Discussion by syedshadab19** [squidfunk/mkdocs-material#4235] [Q&A] [answered]
Hi I am running this command "docker run --rm -it -p 8000:8000 -v ${PWD}:/docs squidfunk/mkdocs-material" on ubuntu/linux server to run the docker and I am getting the below error.

INFO - Building documentation...
Error: Config file 'mkdocs.yml' does not exist.

**Comment by squidfunk** [comment, marked as answer]
Please make sure the file actually exists in the folder you're running the Docker command from.

**Reply by syedshadab19** [reply]
the file exists if i run the same command in windows docker "docker run --rm -it -p 8000:8000 -v %cd%:/docs squidfunk/mkdocs-material" by changing ${pwd} to %cd% the image runs fine. but in linux/ubuntu the command is not working

**Reply by squidfunk** [reply]
Try replacing `${PWD}` with the absolute path to the directory where your `mkdocs.yml` resides.

**Reply by syedshadab19** [reply]
ok thank you it worked
